services/gateway/config.js

```diff
@@ -111,4 +111,8 @@ config.job = {
 	},
 };
 
+config.cors = {
+	allowedOrigin: '*',
```

Comment:
I think we also have to allow users to configure the value:
```suggestion
	allowedOrigin: process.env.CORS_ALLOWED_ORIGIN || '*',
```